src/test/resources/automation.xml

```diff
@@ -0,0 +1,268 @@
+<!--
+
```

Comment:
We might need to change these license headers. Please check the other files too.

Reply:
Updated license headers